compiler/src/iree/compiler/Codegen/Passes.h

```diff
@@ -150,7 +150,7 @@ createTestPartitionableLoopsInterfacePass();
 
 /// Pass to tile and distribute to workgroups.
 std::unique_ptr<OperationPass<IREE::HAL::ExecutableVariantOp>>
-createTileAndDistributeToWorkgroupsPass();
+createTileAndDistributeToWorkgroupsPass(int32_t maxWorkgroupParallelDims = 3);
```

Comment:
Please dont hard code to `3`... use `kNumMaxParallelDims`.

Reply:
It is still hard-coded to 3....

Reply:
fixed for real